LaunchDarkly.podspec

```diff
@@ -39,5 +39,8 @@ Pod::Spec.new do |ld|
 
   ld.subspec 'Core' do |es|
     es.dependency 'LDSwiftEventSource', '3.1.1'
+    es.ios.dependency 'GZIP', '1.3.2'
```

Comment:
This gzip library doesn't support watchos. Which is probably reasonable since CPU cycles are going to be more precious on that device.